modules/azure/vault/vault.tf

```diff
@@ -14,6 +14,11 @@ resource "azurerm_virtual_machine" "vault" {
   network_interface_ids = ["${var.network_interface_ids[0]}"]
   vm_size               = "${var.vm_size}"
 
+  identity =
+  {
+    type = "SystemAssigned"
+  }
```

Comment:
Will it work on german cloud (if we still support it)?

Reply:
Nope. But I'll add conditionals for aws china and azure german/stack in separate PR